lucet-module/src/bindings.rs

```diff
@@ -1,4 +1,4 @@
-use failure::{format_err, Error};
+use anyhow::Error;
```

Comment:
You've done a fine job of converting this module over, but I think it's probably time to convert these errors to new variants on the enum in lucet-module/src/error.rs. They're all caused by bad input, as opposed to "impossible" cases that would be bugs if we hit in normal use, which means it'd be nice to be able to get more than a string from them.

Reply:
Got it!  Will do.